MaxText/inference_mlperf/README.md

```diff
@@ -14,13 +14,22 @@ source .env/bin/activate
 
 ### Install loadgen
 ```
-sudo apt-get install python3-dev
-sudo apt-get install build-essential -y
-git clone git@github.com:mlcommons/inference.git
-cd inference/
-cd loadgen/ && python3 -m pip install .
+pip install absl-py numpy
```

Comment:
Nit: add a comment saying this is taken from the mlperf readme as of (May 2025).